articles/cms/wordpress/configuration.md

```diff
@@ -1,56 +1,65 @@
+1. Login to the [Auth0 dashboard](${uiURL}). (If you don't already have an Auth0 account, you can [create one](https://auth0.com).
+2. Navigate to the *Apps / APIs* page and click **+ New App / API**.
+3. In the *Create Application* window, name your app and click **Save**.
 
-#### New App
+### Get your *Domain*, *Client Id* and *Client Secret*
 
-1. While logged in as an administrator of your WordPress installation, in the **Plugins** section of the dashboard, click *Settings* under *WordPress Auth0 Integration* to open the *Auth0 Settings* page. (If you don't already have an Auth0 account, you can [create one](https://auth0.com).)
-2. On the *Auth0 Settings* page, click *create an application*.
-3. In the new browser window select *+ New App / API*.
-4. On the *Apps / APIs* page, name the new app and click *Save*.
-5. On the new app's *Quick Settings* page, click *Settings*.
-6. From the *Application Settings* page on Auth0, copy the *Auth0 Domain*, *Client Id* and *Client Secret* to the *Auth0 Settings* page of the WordPress dashboard.
-7. Click *Save Changes* at the bottom of the page.
+1. Go to the [Apps / APIs](${uiURL}/#/applications) of the Auth0 dashboard and select the app you want to connect to WordPress.
+2. Leave this browser window open.
+
+### Copy your *Domain*, *Client Id* and *Client Secret*
+
+1. In a separate browser window, login as an administrator of your WordPress installation.
+2. Click on **Auth0** in the left menu of the WordPress dashboard and select **Settings**.
+3. Select the *Basic* settings tab of the *Auth0 Settings* page of the plugin.
+4. Copy the *Domain*, *Client Id* and *Client Secret* settings from the *Settings* page of your app in the Auth0 dashboard to the *Auth0 Settings > Basic* page of your WordPress account.
+5. Click **Save Changes** at the bottom of the page.
 
 ![](/media/articles/cms/wordpress/wp-auth0-initial-config.gif)
 
-## Settings
+## *Auth0 for WordPress* Plugin Settings
 
 ### Basic
 
-- **Domain:** The app domain copied from the app settings in your dashboard.
-- **Client Id:** The app client id copied from the app settings in your dashboard.
-- **Client Secret:** The app client secret copied from the app settings in your dashboard.
-- **WordPress login enabled:** Displays a link on the login page to access the regular WordPress login.
+* **Domain:** The app domain copied from the app settings in your dashboard.
+* **Client Id:** The app client id copied from the app settings in your dashboard.
+* **Client Secret:** The app client secret copied from the app settings in your dashboard.
+* **App token:** The token required to allow the plugin to communicate with Auth0 to update your account settings. If the token has been set, this field will display "Not Visible". If blank, no token has been provided and you will have to [generate a token](/api/v2) with the appropriate scopes, as listed here.
```

Comment:
They are always the same, but I wouldn't add them here, just in case it change sometime, so we avoid to have through all the docs to update them. I would just mention to check the plugin settings for the entire list